packages/mask/src/social-network-adaptor/twitter.com/utils/selector.ts

```diff
@@ -95,12 +95,17 @@ export const sideBarProfileSelector: () => LiveSelector<E, true> = () =>
     querySelector<E>('[role="banner"] [role="navigation"] [aria-label="Lists"] > div')
 export const postEditorInTimelineSelector: () => LiveSelector<E, true> = () =>
     querySelector<E>('[role="main"] :not(aside) > [role="progressbar"] ~ div [role="button"][aria-label]:nth-child(6)')
+
+export const isReply = () => !!location.pathname.match(/^\/\w+\/status\/\d+$/)
 export const postEditorDraftContentSelector = () => {
     if (location.pathname === '/compose/tweet') {
         return querySelector<HTMLDivElement>(
             '[contenteditable][aria-label][spellcheck],textarea[aria-label][spellcheck]',
         )
     }
+    if (isReply()) {
```

Comment:
Here it is detecting whether a reply editor is used with the current location. Does it go to mismatch if the composition modal is opening? Like the pic below, both two editors exist.

<img width="591" alt="image" src="https://user-images.githubusercontent.com/52657989/159483370-d1a8baaf-3b6b-47e3-832d-2651b3d6adaa.png">

Reply:
@lelenei